index.html

```diff
@@ -185,49 +260,74 @@ <h3>Skills</h3>
       </section>
 
       <footer class="form-section">
-        <h2 class="form-desc">
-          I'm always interested in hearing about new projects, so if you'd like
-          to chat please get in touch.
-        </h2>
-
-        <form
-          action="https://formspree.io/f/xbjwrppl"
-          method="POST"
-          id="my-form"
-        >
-          <input
-            type="text"
-            id="name"
-            name="user_name"
-            placeholder="Full Name"
-            maxlength="30"
-            required
-          />
+        <div class="form-a">
+          <h2 class="form-desc">
+            I'm always interested in hearing about new projects, so if you'd
+            like to chat please get in touch.
+          </h2>
 
-          <input
-            type="email"
-            id="email"
-            placeholder="Email address"
-            name="user_email"
-            required
-          />
+          <form
+            action="https://formspree.io/f/xbjwrppl"
+            method="POST"
+            id="my-form"
+          >
+            <input
+              type="text"
+              id="name"
+              name="user_name"
+              placeholder="Full Name"
+              maxlength="30"
+              required
+            />
 
-          <textarea
-            name="message"
-            id="msg"
-            placeholder="Enter text here"
-            maxlength="500"
-            required
-          ></textarea>
+            <input
+              type="email"
+              id="email"
+              placeholder="Email address"
+              name="user_email"
+              required
+            />
 
-          <button type="submit" class="btn btn-center">Get In touch</button>
-        </form>
+            <textarea
+              name="message"
+              id="msg"
+              placeholder="Enter text here"
+              maxlength="500"
+              required
+            ></textarea>
 
+            <button type="submit" class="btn btn-center">Get In touch</button>
+          </form>
+        </div>
         <hr />
-        <div class="icon">
-          <a href="#"
-            ><img src="src/Social-Media-dark.svg" alt="social-media" />
-          </a>
+        <div class="icon-form">
+          <ul>
+            <li>
+              <a href="#"
```

Comment:
- [ ] (Optional) I think it would be nice to use a `margin-top: 25px` for this section. That way, your design would like that of Figma's template that you chose.